src/collectors/windows.plugin/windows_plugin.c

```diff
@@ -30,6 +30,11 @@ static struct proc_module {
      .enabled = CONFIG_BOOLEAN_YES,
      .update_every = UPDATE_EVERY_MIN,
      .func = do_GetPowerSupply},
+    {.name = "PerflibServices",
+     .dim = "PerflibServices",
+     .enabled = CONFIG_BOOLEAN_YES,
+     .update_every = 30 * UPDATE_EVERY_MIN,
```

Comment:
Do you think this will work? Did you try it? Given the 15-minute default timeframe, the chart will likely be empty 50% of the time due to data collection delays exceeding 15 minutes.